lib/mysql/index.js

```diff
@@ -297,6 +297,12 @@ const findById = R.curry((mysql, table, projection, id) =>
 )
 
 
+// Permenently delete a record from given table.
+const deleteById = R.curry((mysql, table, id) =>
+  Query.deleteById(mysql, table, id)
+)
```

Comment:
this should throw a NotFound error like the other stores if the id does not exist.